spring-cloud-starter-dataflow-server-local/src/test/resources/org/springframework/cloud/dataflow/server/local/security/oauthConfig.yml

```diff
@@ -1,6 +1,3 @@
-management:
-  security:
-    enabled: true
```

Comment:
I see that this is removed as it requires `ADMIN` role (from spring-boot) to access the management actuator endpoints. I understand this is only in tests, but, what happens when `management.security.enabled=true` at the server level?